spark-tsfile/src/main/scala/org/apache/iotdb/tsfile/package.scala

```diff
@@ -19,15 +19,24 @@
 
 package org.apache.iotdb
 
-import org.apache.spark.sql.{DataFrame, DataFrameReader, DataFrameWriter}
+import org.apache.spark.sql.{DataFrame, DataFrameReader, DataFrameWriter, SparkSession}
 
 package object tsfile {
 
   /**
     * add a method 'tsfile' to DataFrameReader to read tsfile
+    * @param reader dataframeReader
     */
   implicit class TsFileDataFrameReader(reader: DataFrameReader) {
-    def tsfile: String => DataFrame = reader.format("org.apache.iotdb.tsfile").load
+    def tsfile(path: String,
+               isNewForm: Boolean = false): DataFrame = {
+      if (isNewForm) {
+        reader.option(DefaultSource.path, path).option(DefaultSource.isNewForm, "new_form").format("org.apache.iotdb.tsfile").load
```

Comment:
the parameter could be a boolean

Reply:
in scala, Boolean is not a class but a primitive type